.github/workflows/Build.yaml

```diff
@@ -59,6 +56,65 @@ jobs:
           name: lint-reports
           path: '**/build/reports/lint-results-*.html'
 
+  test:
```

Comment:
Shouldn't we also add `needs: build`?

Reply:
No, that would delay the execution of this step and we want quick feedback over lower use of resources.